src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
+        this.side = side;
+        this.setColor(color);
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Drawing a square");
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public String getFigureData() {
```

Comment:
The `getFigureData()` method should be refactored to use `String.format` for better readability and to avoid concatenation in `System.out.println()` or similar statements.